src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+ 
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+ 
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.ConfigureMetadataPage;
+import org.openmrs.contrib.qaframework.page.OpenConceptLabPage;
+import org.openmrs.contrib.qaframework.page.OpenConceptLabSuccessPage;
+import org.openmrs.contrib.qaframework.page.SubscriptionPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+ 
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+ 
+public class OpenConceptLabSteps extends Steps  {
+    private OpenConceptLabPage openconceptlabpage;
+    private ConfigureMetadataPage configureMetadataPage;
+    private static final String releasedDictionaryUrl = "https://api.staging.openconceptlab.org/users/openmrs/collections/TD-ecly/65725685/";
+    private static final String tokenUrl = "bd022mad6d3df24f5c42ewewa94b53a23edf6eee7r";
+    private static final String newDictionaryUrl = "http://api.openconceptlab.com/orgs/CIEL/sources/CIEL/";
+    private static final String newTokenUrl = "bd022mad6d3df24f5c42ewewa94b53a23edf6fff7r";
+    private OpenConceptLabSuccessPage openConceptLabSuccessPage;
+    private OpenConceptLabPage openConceptLabPage;
+    private SubscriptionPage subscriptionPage;
+    
+    @Before(RunTest.HOOK.SELENIUM_OPEN_CONCEPT_LAB)
+    public void visitHomePage() {
+        initiateWithLogin();    
+    }
+    
+    @After(RunTest.HOOK.SELENIUM_OPEN_CONCEPT_LAB)
+    public void destroy(){
+        quit();
+    }
+    
+    @Given("a user clicks on configure Metadata link from home page")
+    public void launchMetadataDashboard() {
+    	configureMetadataPage = (ConfigureMetadataPage) homePage.goToConfigureMetadata();
+    }
+    
+    @When("the system loads the configure metadata page")
+    public void systemLoadsConfigureMetadataPage() {
+    	assertPage(configureMetadataPage.waitForPage());
+    }
+ 
+    @And("a user clicks Manage OCL link from configure metadata page")
+    public void loadOpenConceptLabPage() {
+        openconceptlabpage = (OpenConceptLabPage) configureMetadataPage.goToOpenConceptLabPage();
+    }
+    
+    @Then("the system loads Open Concept Lab page")
```

Comment:
@jwnasambu is this gherkin step properly corresponds to the one written in the feature file?
```suggestion
    @Then("the system loads Open Concept Lab page")
```

Reply:
@kdaud you are right! am gonna fix it. Oh my head is over saturated.

Reply:
@sherrif10 @kdaud am the ticket is closed again? am sorry am not okay health wise and am trying to go slow. Kindly bare with me.

Reply:
> 

Sorry about that , quick recovery

Reply:
@jwnasambu please take your time to recover and do not fell pressured over the ticket your working on. You can resume work on it when you get well and ready to work on it.

Reply:
> the ticket is closed again

That ticket was closed as a result of the ticket-label that could not make it possible to move it from one state to another. However, @sherrif10 created [RATEST-275](https://issues.openmrs.org/browse/RATEST-275) which you need to claim so that no body duplicates your work.